Fix for compilation error with flutter version 3.29

rahulmaindargi opened the pull request:
https://github.com/mannprerak2/nearby_connections/issues/83

rahulmaindargi committed:
Implemented service aware

rahulmaindargi committed:
import fix

rahulmaindargi committed:
fix for FlutterJNI( 7138): Tried to send a platform message to Flutter, but FlutterJNI was detached from native C++. Could not send. Channel: nearby_connections. Response ID: 4
Refer from.
https://medium.com/@gauravpaudel2013/tried-to-send-a-platform-message-to-flutter-but-flutterjni-was-detached-from-native-c-well-well-c7d9ccf0df96

rahulmaindargi committed:
fix for FlutterJNI( 7138): Tried to send a platform message to Flutter, but FlutterJNI was detached from native C++. Could not send. Channel: nearby_connections. Response ID: 4
Refer from.
https://medium.com/@gauravpaudel2013/tried-to-send-a-platform-message-to-flutter-but-flutterjni-was-detached-from-native-c-well-well-c7d9ccf0df96

rahulmaindargi committed:
fix for FlutterJNI( 7138): Tried to send a platform message to Flutter, but FlutterJNI was detached from native C++. Could not send. Channel: nearby_connections. Response ID: 4
Refer from.
https://medium.com/@gauravpaudel2013/tried-to-send-a-platform-message-to-flutter-but-flutterjni-was-detached-from-native-c-well-well-c7d9ccf0df96

rahulmaindargi committed:
fix for FlutterJNI( 7138): Tried to send a platform message to Flutter, but FlutterJNI was detached from native C++. Could not send. Channel: nearby_connections. Response ID: 4
Refer from.
https://medium.com/@gauravpaudel2013/tried-to-send-a-platform-message-to-flutter-but-flutterjni-was-detached-from-native-c-well-well-c7d9ccf0df96

rahulmaindargi committed:
fix for FlutterJNI( 7138): Tried to send a platform message to Flutter, but FlutterJNI was detached from native C++. Could not send. Channel: nearby_connections. Response ID: 4
Refer from.
https://medium.com/@gauravpaudel2013/tried-to-send-a-platform-message-to-flutter-but-flutterjni-was-detached-from-native-c-well-well-c7d9ccf0df96

rahulmaindargi committed:
fix for FlutterJNI( 7138): Tried to send a platform message to Flutter, but FlutterJNI was detached from native C++. Could not send. Channel: nearby_connections. Response ID: 4
Refer from.
https://medium.com/@gauravpaudel2013/tried-to-send-a-platform-message-to-flutter-but-flutterjni-was-detached-from-native-c-well-well-c7d9ccf0df96

rahulmaindargi committed:
fix for FlutterJNI( 7138): Tried to send a platform message to Flutter, but FlutterJNI was detached from native C++. Could not send. Channel: nearby_connections. Response ID: 4
Refer from.
https://medium.com/@gauravpaudel2013/tried-to-send-a-platform-message-to-flutter-but-flutterjni-was-detached-from-native-c-well-well-c7d9ccf0df96

rahulmaindargi committed:
fix for FlutterJNI( 7138): Tried to send a platform message to Flutter, but FlutterJNI was detached from native C++. Could not send. Channel: nearby_connections. Response ID: 4
Refer from.
https://medium.com/@gauravpaudel2013/tried-to-send-a-platform-message-to-flutter-but-flutterjni-was-detached-from-native-c-well-well-c7d9ccf0df96

rahulmaindargi committed:
fix for FlutterJNI( 7138): Tried to send a platform message to Flutter, but FlutterJNI was detached from native C++. Could not send. Channel: nearby_connections. Response ID: 4
Refer from.
https://medium.com/@gauravpaudel2013/tried-to-send-a-platform-message-to-flutter-but-flutterjni-was-detached-from-native-c-well-well-c7d9ccf0df96

rahulmaindargi committed:
fix for FlutterJNI( 7138): Tried to send a platform message to Flutter, but FlutterJNI was detached from native C++. Could not send. Channel: nearby_connections. Response ID: 4
Refer from.
https://medium.com/@gauravpaudel2013/tried-to-send-a-platform-message-to-flutter-but-flutterjni-was-detached-from-native-c-well-well-c7d9ccf0df96

rahulmaindargi committed:
fix for FlutterJNI( 7138): Tried to send a platform message to Flutter, but FlutterJNI was detached from native C++. Could not send. Channel: nearby_connections. Response ID: 4
Refer from.
https://medium.com/@gauravpaudel2013/tried-to-send-a-platform-message-to-flutter-but-flutterjni-was-detached-from-native-c-well-well-c7d9ccf0df96

rahulmaindargi committed:
fix for FlutterJNI( 7138): Tried to send a platform message to Flutter, but FlutterJNI was detached from native C++. Could not send. Channel: nearby_connections. Response ID: 4
Refer from.
https://medium.com/@gauravpaudel2013/tried-to-send-a-platform-message-to-flutter-but-flutterjni-was-detached-from-native-c-well-well-c7d9ccf0df96

rahulmaindargi committed:
changes for event channel

rahulmaindargi committed:
changes for event channel

rahulmaindargi committed:
Refactor: Remove Legacy Plugin Registration and Unused Code in NearbyConnectionsPlugin

- Removed the legacy plugin registration method `registerWith` as it is no longer needed.
- Removed unused `pluginRegistrar` variable.
- Eliminated commented-out code related to method invocation within event callbacks for better readability.
- Replaced the `//channel.invokeMethod` with `//` for readability in `onConnectionInitiated`, `onConnectionResult`, `onDisconnected`, `onPayloadReceived`, `onPayloadTransferUpdate`, `onEndpointFound`, and `onEndpointLost` callbacks.

rahulmaindargi commented on lib/src/nearby.dart:
Not sure why it is coming as a changed file. 
I am unable to find a way to remove the file from Pull request, but its not required to fix compilation error

mannprerak2 replied:
you can try running git pull upstream release --no-rebase

then resolve the merge conflicts (use keep theirs)

rahulmaindargi replied:
Done.

rahulmaindargi committed:
Merge remote-tracking branch 'OriginFork/master'

# Conflicts:
#	android/src/main/java/com/pkmnapps/nearby_connections/NearbyConnectionsPlugin.java
#	lib/src/nearby.dart

mannprerak2 commented:
u may wanna run `dart format .` for the tests to pass

rahulmaindargi committed:
Refactor: Align Dart code within the `Nearby` class

This commit refactors the Dart code within the `Nearby` class to improve readability by aligning the code within the `receiveBroadcastStream` listener. Specifically, it ensures consistency by aligning the `case` statements and their corresponding code blocks.

mannprerak2 commented:
Thanks @rahulmaindargi 🚀
